### PR TITLE
use FindClustersAndDimRedux instead etc

### PR DESCRIPTION
**Pull request by ok-gitr**

### CODE REVIEW

File: R/postSDA_processing.R [hunk 1]
```diff
@@ -13,52 +13,64 @@
...
+                             #scale.factor = 10000,
+                             verbose = T)
   
-  #TODO: if the imputed is pre-normalized is this redundant?
-  seuratObj <- NormalizeData(seuratObj, normalization.method = "LogNormalize", scale.factor = 10000)
-  tempData <- as.data.frame(seuratObj@assays$RNA@data)
-  tempData[is.na(tempData)] <- 0
-  tempData <- as(as.matrix(tempData), "dgCMatrix")
-  seuratObj@assays$RNA@data <- tempData
+  #drop NaN values in matrix (otherwise DE test errors)
```
Comment (bbimber) [inline R/postSDA_processing.R:42]: @kolabx why not do this on the recomposedMat, before creating the seurat object?  Do we know whether NAs are a problem for NormalizeData?

Comment (bbimber) [reply R/postSDA_processing.R:42]: sorry, i see this is data, not counts.  more to the point, do you know why NormalizeData is giving NAs on this input?  is that the real problem?

Comment (ok-gitr) [reply R/postSDA_processing.R:42]: @bbimber yes, the dot product of gene loadings and cell scores to recompose the matrix produces negative values. when these are logNormalized they produce NaN values

Comment (eisascience) [reply R/postSDA_processing.R:42]: I have a new update to deal with this, thus making this branch moot.

**Pull request closed without merging**